test/parallel/test-domain-uncaught-exception.js

```diff
@@ -23,7 +23,7 @@ function test1() {
    */
   const d = domain.create();
   d.run(function() {
-    setTimeout(function onTimeout() {
+    setImmediate(function onSetImmediate() {
```

Comment:
I understand your point. Since it does not specify a duration (the second argument), I think it's more appropriate to change this call to `setImmediate()` instead of `setTimeout()`.